src/scripts.js

```diff
@@ -83,6 +89,30 @@ const findRecipe = (recipeId, cookBook) => {
   viewRecipe()
 }
 
+const favoriteStore = (recipeId, cookBook) => {
```

Comment:
These look like a good opportunity for refactoring in the future. Happy to see it working for now :)